std/string.d

```diff
@@ -193,12 +193,13 @@ public import std.uni : icmp, toLower, toLowerInPlace, toUpper, toUpperInPlace;
 public import std.format : format, sformat;
 import std.typecons : Flag;
 
-import std.meta;
-import std.range.primitives;
-import std.traits;
+import std.meta;// AliasSeq, staticIndexOf
+import std.range.primitives;// ElementEncodingType, ElementType, hasLength, hasSlicing, isBidirectionalRange, isForwardRange, isInputRange, isOutputRange, isRandomAccessRange
+import std.traits;// isConvertibleToString, isNarrowString, isSomeChar, isSomeString, StringTypeOf, Unqual
 
```

Comment:
>That, and there's no space between ; and //, and the comments should be aligned if possible.

There's other instances without the space in phobos, but whatever, I'll add it and break up the lines.  As for alignment, these aren't really comments, so shouldn't be a need.

>Why do they need to exist at all?

The plan is to remove the comment slashes and turn these into selective imports, once we have the green light for module-scope selective imports in phobos, after the current transitional period to accomodate downstream users is over.